bookwyrm_backend/src/main/java/com/bookwyrm/backend/review/input/ReviewUploadInput.java

```diff
@@ -1,10 +1,16 @@
 package com.bookwyrm.backend.review.input;
 
+import com.bookwyrm.backend.comment.objects.Comment;
+
+import java.util.List;
+
 public class ReviewUploadInput {
 
     private String id;
+    private String title;
     private String author;
-    private String description;
+    private String content;
+    private List<Comment> commentList;
```

Comment:
Apologies, I believe I misunderstood when suggesting this. Reviews should never be created with pre-existing reviews, so we can leave the comment list out for the input.

Reply:
No problem, I have removed the comment object file and also removed commentList from the review endpoint files.